docs/how-to/delete-an-ensembler/README.md

```diff
@@ -0,0 +1,7 @@
+# Deleting Ensembler
+
+An ensembler that is used by router version or ensembling jobs can be deleted. An ensembler only can be deleted if it is not in active use in routers or ensembling jobs. Deleting an ensembler will result in the purging of the related router version, ensembling job, and ensembler metadata from turing database. This action is **irreversible**.
```

Comment:
hmm at first i just want to tell that the ensembler could be deleted, but yeah it kinda create a confusion, let me change the sentences so it will be clearer